dist/images/install.sh

```diff
@@ -2023,6 +2023,10 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
```

Comment:
should be able to set ENABLE_BIND_LOCAL_IP from install.sh and helm

Reply:
I set ENABLE_BIND_LOCAL_IP = true, on another pr , and it seems ok when running workflows

Reply:
This option should be configurable

Reply:
it can be on/off, the pod will restart using a new env value.

Reply:
fixed